server/store/user/user.go

```diff
@@ -85,3 +87,16 @@ func (s userStore) AdminExists() bool {
 	var user core.User
 	return !s.db.Where("role = ?", "admin").First(&user).RecordNotFound()
 }
+
+func HumanizeError(err error) error {
```

Comment:
there is no need to export that function, it should start with lowercase letter imho.

Reply:
You're right, fixed